concepts/05 Widgets/DataGrid/35 Paging/Paging.md

```diff
@@ -40,6 +40,58 @@ Paging is used to load data in portions, which improves the widget's performance
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ... >
+            <DxPaging
+                :page-size="15"
+                :page-index="1" /> <!-- Shows the second page -->
+        </DxDataGrid>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDataGrid, {
+        DxPaging
+    } from 'devextreme-vue/data-grid';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxPaging
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DataGrid, {
+        Paging
+    } from 'devextreme-react/data-grid';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <DataGrid ... >
+                    <Paging
+                        pageSize={15}
+                        pageIndex={1} /> {/* Shows the second page */}
```

Comment:
Use defaultPageIndex/defaultPageSize or define state for pageIndex/pageSize and add onOptionChanged handler. Otherwise paging will not work.